modules/contentbox/modules/contentbox-admin/handlers/authors.cfc

```diff
@@ -546,6 +546,10 @@ component extends="baseHandler"{
 	* Change user password
 	*/
 	function passwordChange( event, rc, prc ){
+		if( prc.oCurrentAuthor.getAuthorID() != rc.authorID ){
+			cbMessagebox.error( "You cannot change passwords for other users. Please start a password reset instead." );
+			return setNextEvent(event=prc.xehAuthorEditor, queryString="authorID=#rc.authorID#" );
```

Comment:
spacing on `(` please

Reply:
spacing still needed

Reply:
It shows updated in my code. Re-pushing.

Reply:
It shows if you look at the full file in the branch 
https://github.com/Ortus-Solutions/ContentBox/blob/CONTENTBOX-970-GP/modules/contentbox/modules/contentbox-admin/handlers/authors.cfc